packages/react-components/react-combobox/src/utils/useComboboxPopup.ts

```diff
@@ -33,13 +19,11 @@ export function useComboboxPopup(
     align: 'start' as const,
     offset: { crossAxis: 0, mainAxis: 2 },
     fallbackPositions,
+    matchTargetSize: 'width' as const,
     ...resolvePositioningShorthand(positioning),
   };
 
   const { targetRef, containerRef } = usePositioning(popperOptions);
 
-  const listboxRef = useMergedRefs(listboxShorthand?.ref, containerRef);
-  const listbox = listboxShorthand && { ...listboxShorthand, ref: listboxRef };
-
-  return [{ ...triggerShorthand, ref: useMergedRefs(triggerShorthand?.ref, targetRef) }, listbox];
+  return [containerRef, targetRef];
```

Comment:
Did some refactoring here, since all this hook was doing was assigning refs. The result should be a bit simpler

Reply:
Should be hook renamed then? `useComboboxPopupPositioning`?

Reply:
yeah actually probably, `useComboboxPositioning` would be my choice